src/mrpro/data/_TrajectoryDescription.py

```diff
@@ -25,18 +25,18 @@ class TrajectoryDescription:
     """TrajectoryDescription dataclass."""
 
     identifier: str = ''
-    userParameterLong: dict[str, int] = dataclasses.field(default_factory=dict)
-    userParameterDouble: dict[str, float] = dataclasses.field(default_factory=dict)
-    userParameterString: dict[str, str] = dataclasses.field(default_factory=dict)
+    user_parameter_long: dict[str, int] = dataclasses.field(default_factory=dict)
```

Comment:
userParameterLong is the name used by ismrmrd...
should we consider keeping it and marking this with a` #noqa`  comment to disable this rele here?

Reply:
or do we always follow pep8?
not sure which one I prefer.

Reply:
IMO we should always follow pep8 except for a few exceptions such as LinearOperator.H, which we mark with `# noqa: N812`

Reply:
I would go for user_parameter_long. I think it is easy enough to figure out that this corresponds with userParameterLong in ISMRMRD.